extensions/Worker.Extensions.ServiceBus/src/ServiceBusExtensionStartup.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+public class ServiceBusExtensionStartup : WorkerExtensionStartup
```

Comment:
For this startup class code to be included in the startup process, we should use the assembly level attribute `WorkerExtensionStartup` as well. (Step 3 [here](https://github.com/Azure/azure-functions-dotnet-worker/wiki/Extension-startup-hook))